hyperloglog.go

```diff
@@ -81,3 +83,49 @@ func (h *HyperLogLog) Count() uint64 {
 	}
 	return uint64(-two32 * math.Log(1-est/two32))
 }
+
+// BinaryMarshaler marshals the HyperLogLog into binary form for storage
+func (h *HyperLogLog) BinaryMarshaler() ([]byte, error) {
```

Comment:
I think this should be called MarshalBinary, and the other called UnmarshalBinary to implement the BinaryMarshaler and BinaryUnmarshaler interface

http://golang.org/pkg/encoding/